src/tracker/tracker-funcs.jl

```diff
@@ -497,3 +497,48 @@ function addfloemasks!(props, imgs)
     end
     return nothing
 end
+
+"""
+    process_soit(passtimesdir::String)
+
+Process the [SOIT](https://github.com/WilhelmusLab/SOIT-Satellite-Overpass-Identification-Tool) output file.
+
+The SOIT output file is a csv file with columns :Date, :sat1, :sat2, ... :satn, where each row is a pass time for the satellites listed in the columns. The resulting DataFrame has columns `:sat` and `:pass_time` sorted by :pass_time.
+
+# Arguments
+- `passtimesdir::String`: The directory containing the SOIT output file
+"""
+function process_soit(passtimesdir::String)
+    # check there is a file at passtimesdir starting with "passtimes_lat" with extension .csv
+    pth = filter(x -> occursin(r"^passtimes_lat.*\.csv$", x), readdir(passtimesdir))
+    isempty(pth) && error("No csv file found at $passtimesdir starting with 'passtimes_lat'")
+    length(pth) > 1 && error("More than one csv file found at $passtimesdir starting with 'passtimes_lat'")
```

Comment:
Should we handle the multiple soit files during param cycling? The results structure looks different when expanding parameters: 
![image](https://github.com/WilhelmusLab/IceFloeTracker.jl/assets/24841909/31e8f554-b270-4019-b8e7-8754ddfd1f42)

Reply:
@tdivoll This is now handled by cycl. I think this is okay to merge?